bindings/pydairlib/common_py.cc

```diff
@@ -0,0 +1,25 @@
+#include <pybind11/eigen.h>
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "common/find_resource.h"
+
+namespace py = pybind11;
+
+namespace dairlib  {
+namespace pydairlib {
+
+PYBIND11_MODULE(common, m) {
```

Comment:
Is this file supposed to be committed? It looks like a duplicate of the one in `common/` and is not reference in the main BUILD.bazel